docs/05_features/10_resource_payer.md

```diff
@@ -0,0 +1,46 @@
+---
+content_title: Resource Payer
+---
+
+## Overview
+
+The `Resource Payer` feature, also known as `Transaction Sponsorship`, allows to specify the resource payer for a transaction. As a direct consequence the application developers have a simple and secure way to sponsor, or decline to sponsor, transactions on behalf of their users.
```

Comment:
Looks good overall! Just a comment about not enclosing the feature name in back ticks. 

Change `Resource Payer` to Resource Payer in all instances when mentioning the feature name.